src/sql/workbench/common/editor/query/queryEditorInput.ts

```diff
@@ -328,7 +303,7 @@ export abstract class QueryEditorInput extends EditorInput implements IConnectab
 		this.state.connecting = false;
 	}
 
-	public onConnectSuccess(params?: INewConnectionParams): void {
+	public async onConnectSuccess(params?: INewConnectionParams): Promise<void> {
```

Comment:
This shouldn't need to be async since you aren't awaiting the contextualizeEditorForCopilot call